components: (google-cloud-sdk) Add supported OP names and description to readme. (#5487)
kubeflow/pipelines: components/google-cloud/README.md

    #endpoint
    model_deploy_step.outputs['endpoint_metadata']
    ```

    ### Supported Components 
    Following is the list of currently supproted components. 

    ```python
    AutoMLImageTrainingJobRunOp(...)

>         """Runs the AutoML Image training job and returns a model."""

Reviewer: ```suggestion
        """Runs an AutoML Image training job and returns a model."""
```

    AutoMLTabularTrainingJobRunOp(...)

>         """Runs the training job and returns a model."""

Reviewer: ```suggestion
        """Runs an AutoML Tabular training job and returns a model."""
```

    AutoMLTextTrainingJobRunOp(...)

>         """Runs the training job and returns a model."""

Reviewer: ```suggestion
        """Runs an AutoML Text training job and returns a model."""
```

    AutoMLVideoTrainingJobRunOp(...)

>         """Runs the AutoML Image training job and returns a model."""

Reviewer: ```suggestion
        """Runs an AutoML Image training job and returns a model."""
```

    CustomContainerTrainingJobRunOp(...)

>         """Runs the custom training job."""

Reviewer: ```suggestion
        """Runs a custom container training job."""
```

    CustomPythonPackageTrainingJobRunOp(...)

>         """Runs the custom training job."""

Reviewer: ```suggestion
        """Runs a custom Python package training job."""
```

    EndpointCreateOp(...)
        """Creates a new endpoint."""

    ImageDatasetCreateOp(...)

>         """Creates a new image dataset and optionally imports data into dataset when"""

Reviewer: ```suggestion
        """Creates a new image dataset and optionally imports data into the dataset."""
```

    ImageDatasetExportDataOp(...)
        """Exports data to output dir to GCS."""

    ImageDatasetImportDataOp(...)

>         """Upload data to existing managed dataset."""

Reviewer: ```suggestion
        """Upload Image data to existing managed Image dataset."""
```

    ModelBatchPredictOp(...)

>         """Creates a batch prediction job using this Model and outputs prediction"""

Reviewer: ```suggestion
        """Creates a batch prediction job using this Model and outputs prediction."""
```

    ModelDeployOp(...)
        """Deploys model to endpoint. Endpoint will be created if unspecified."""

    ModelUploadOp(...)
        """Uploads a model and returns a Model representing the uploaded Model resource."""

    TabularDatasetCreateOp(...)
        """Creates a new tabular dataset."""

    TabularDatasetExportDataOp(...)

>         """Exports data to output dir to GCS."""

Reviewer: ```suggestion
        """Exports Tabular data to output dir to GCS."""
```

>     TabularDatasetImportDataOp(...)

Reviewer: This should be removed. Tabular dataset do not support import. I'll remove the op in a follow up PR.

Author: got it, I removed the note from read me and the op from Init.

        """Upload data to existing managed dataset."""

    TextDatasetCreateOp(...)
        """Creates a new text dataset and optionally imports data into dataset when"""

    TextDatasetExportDataOp(...)

>         """Exports data to output dir to GCS."""

Reviewer: ```suggestion
        """Exports Text data to output dir to GCS."""
```

    TextDatasetImportDataOp(...)

>         """Upload data to existing managed dataset."""

Reviewer: ```suggestion
        """Upload data to existing managed Text dataset."""
```

    VideoDatasetCreateOp(...)
        """Creates a new video dataset and optionally imports data into dataset when"""

    VideoDatasetExportDataOp(...)

>         """Exports data to output dir to GCS."""

Reviewer: ```suggestion
        """Exports Video data to output dir to GCS."""
```

    VideoDatasetImportDataOp(...)

>         """Upload data to existing managed dataset."""

Reviewer: ```suggestion
        """Upload data to existing managed Video dataset."""
```
